modules/ossm-config-sidecar-out-mtls.adoc

```diff
@@ -0,0 +1,36 @@
+// Module included in the following assemblies:
+//
+// * service_mesh/v2x/ossm-config.adoc
+
+[id="ossm-security-mtls-sidecars-outgoing_{context}"]
+== Configuring sidecars for outgoing connections
+
+Create a destination rule to configure {ProductShortName} to use mTLS when sending requests to other services in the mesh.
+
+.Procedure
+
+. Create a YAML file using the following example. 
++
+.DestinationRule example destination-rule.yaml
+[source,yaml]
+----
+apiVersion: networking.istio.io/v1alpha3
+kind: DestinationRule
+metadata:
+  name: default
+  namespace: <namespace>
+spec:
+  host: "*.<namespace>.svc.cluster.local"
+  trafficPolicy:
+   tls:
+    mode: ISTIO_MUTUAL
+----
++
+.. Replace `<namespace>` with the namespace where the service is located. 
+
+. Run the following command to upload the resource to the namespace where the service is located. It must match the `namespace` field in the DestinationRule resource you just created
```

Comment:
Based on https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#object-references, I **think** DestinationRule should be `DestinationRule`